dll/win32/shell32/COpenWithMenu.cpp

```diff
@@ -1368,15 +1368,13 @@ COpenWithMenu::Initialize(PCIDLIST_ABSOLUTE pidlFolder,
     if (!_ILIsValue(pidlChild))
     {
         TRACE("pidl is not a file\n");
-        GlobalUnlock(medium.hGlobal);
```

Comment:
So technically, should these still call Unlock because `ReleaseStgMedium` doesn't?

Reply:
ah yes, free also calls unlock but free is not always called.
Thanks.